### PR TITLE
change payload-flattening-threshold from 2 to 1

### PR DESCRIPTION
**Pull request by ankushbindlish2**
payload-flattening-threshold

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sergey-shandar** [comment]
@ankushbindlish2 this's potential SDK breaking changes. Is there a reason to do so?

**Comment by ankushbindlish2** [comment]
It is not. Fixed the indentation, if that created confusion

**Comment by sergey-shandar** [comment]
@shahabhijeet please, review the PR.

**Comment by sergey-shandar** [comment]
@ankushbindlish2 could you also add codeowners for this RP https://github.com/Azure/azure-rest-api-specs/blob/master/CODEOWNERS

**Comment by ankushbindlish2** [comment]
no, not aware of this process...

**Comment by sergey-shandar** [comment]
@ankushbindlish2 could you add couple of people from your team who should be responsile for reviewing changes in `storagesync`.

**Comment by sergey-shandar** [comment]
#3764

**Review by sergey-shandar — APPROVED** [review]

**Comment by sergey-shandar** [comment]
@dsgouda please, review

File: specification/storagesync/resource-manager/readme.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ csharp:
   azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.Azure.Management.StorageSync
-  payload-flattening-threshold: 2
```
Comment (dsgouda) [inline specification/storagesync/resource-manager/readme.md]: `StorageSync` C# SDK has not been published yet, this won't be a breaking change.
@sergey-shandar Please go ahead and merge this

**Pull request merged**